CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -100,6 +100,7 @@ public let KEYWORDS: [KeywordSpec] = [
   KeywordSpec("backDeployed"),
   KeywordSpec("before"),
   KeywordSpec("block"),
+  KeywordSpec("borrowBindingKW", isLexerClassified: true, requiresTrailingSpace: true),
```

Comment:
We shouldn’t be setting `isLexerClassified: true` for any newly introduced keyword because it means that it will be disallowed in some situations that expect an identifier. Function names are one example that come to my mind.

I think everything should just work fine if you remove `isLexerClassified: true` though.